### DISCUSSION TITLE
essay/2023/12/19/todo

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#153] [Comments]
# essay/2023/12/19/todo

일정 관리 일정은 구글캘린더로 관리를 합니다. 2008년도부터 썼던 것 같습니다.

https://jeho.page/essay/2023/12/19/todo.html

**Comment by onestn** [comment]
저도 사용하는 서비스를 운영하는 회사의 주식을 사야겠네요 :)

**Reply by BenjaminKim** [reply]
싼 가격에 사셔야 합니다...!

**Reply by voidFunctionReturn0** [reply]
저도 일정은 구글 캘린더로 관리하고
할일은 마이크로소프트의 Todo 앱으로 합니다.
모바일-PC 동기화가 잘 돼서 좋습니다 ㅎㅎ

**Reply by voidFunctionReturn0** [reply]
실수로 다른 분 댓글에 답글로 작성했네요 ㅎㅎ;
